README.md

```diff
@@ -213,26 +278,26 @@ As already discussed, employees can get restricted stock, stock options, or RSUs
+    - An 83(b) election isn’t guaranteed to reduce your taxes, of course. For example, the value of the stock may not increase. And if you leave the company before you vest, you *don’t* get the taxes you’ve paid back.
     - ❗ You must file the 83(b) election yourself with the IRS [**within 30 days**](https://www.irs.gov/irb/2012-28_IRB/ar12.html) of the grant or exercise, or the opportunity is irrevocably lost.
     - 🔸 Section 83(b) elections cannot be made on the receipt of a stock option. They can only be made on the receipt of actual shares of stock.
-    - If you receive an immediately exercisable stock option (meaning, an option that is early exercisable, when it is not vested), and you exercise the option before the option vests, you can make an 83(b) election on your receipt of the shares on exercise.
-    - Section 83(b) elections do not apply to vested shares; the election only applies to stock that is not yet vested. Thus, if you receive options that are *not* early exercisable, which you cannot exercise until vested, then an 83(b) election would not apply.
+    - If you receive an early exercisable stock option (when you don’t have to wait for the the stock to vest), you can make an 83(b) election on receipt of the shares on exercise.
+    - Section 83(b) elections do not apply to vested shares; the election only applies to stock that is not yet vested. Thus, if you receive options that are *not* early exercisable (meaning you have to wait until they vest to exercise), an 83(b) election would not apply.
     - 🔹 Founders and very early employees will almost always want to do an 83(b) election upon the receipt of unvested shares, since the stock value is probably low. If the value is really low, and the taxes owed are not that great, you can make the election without having to pay much tax and start your capital gains holding period on the shares.
 
 ### 409A valuations
 
 - When stock vests, or you exercise an option, the IRS will consider what the **fair market value (FMV)** of the stock is when determining the tax you owe.
 - Of course, if no one is buying and selling stock, as is the case in most startups, then its value isn’t obvious. For the IRS to evaluate how much stock is worth, it uses what is known as the [**409A valuation**](http://www.fenwick.com/FenwickDocuments/409_Valuations_Stock_Options.pdf) of the company.
-    - The startup pays for an appraisal that sets the 409A, typically annually or after events like fundraising. In practice, this number could be low or high.
+    - The startup pays for an appraisal that sets the 409A, typically annually or after events like fundraising.
```

Comment:
A 409A does have to happen every 12 months to grant the company safe harbor. A 409A has to be done after any event that could be deemed a "material event," which is a fancy way of saying any event that could change the price or value of the company meaningfully. Other examples could be if a CEO leaves, if the company starts making a ton of money, or an acquisition.

Reply:
🚧